{
  "debianbug": 560755,
  "description": "python-docutils allows insecure usage of temporary files",
  "releases": {
    "bullseye": {
      "fixed_version": "0.6-2",
      "repositories": {
        "bullseye": "0.16+dfsg-3"
      },
      "status": "resolved",
      "urgency": "low"
    },
    "buster": {
      "fixed_version": "0.6-2",
      "repositories": {
        "buster": "0.14+dfsg-4"
      },
      "status": "resolved",
      "urgency": "low"
    },
    "sid": {
      "fixed_version": "0.6-2",
      "repositories": {
        "sid": "0.16+dfsg-4"
      },
      "status": "resolved",
      "urgency": "low"
    },
    "stretch": {
      "fixed_version": "0.6-2",
      "repositories": {
        "stretch": "0.13.1+dfsg-2"
      },
      "status": "resolved",
      "urgency": "low"
    }
  },
  "scope": "local"
}